release_notes/ocp-4-12-release-notes.adoc

```diff
@@ -143,6 +143,83 @@ In earlier versions of {product-title}, Ironic container images used {op-system-
 
 For more information about the Ironic provisioning service, see xref:../installing/installing_bare_metal_ipi/ipi-install-overview.adoc[Deploying installer-provisioned clusters on bare metal].
 
+[id="ocp-4-12-agent-based-install-asset-input"]
+==== Agent-based installation supports two input modes
+The Agent-based installation supports two input modes:
+
+**Preferred**
+
+* `install-config.yaml` file
+* `agent-config.yaml` file
+
+**Optional**
+
+* Zero Touch Provisioning(ZTP) manifests
+
+With the preferred mode, you can configure the `install-config.yaml` file and specify Agent-based specific settings in the `agent-config.yaml` file.
+For more information, see xref:../installing/installing_with_agent_based_installer/preparing-to-install-with-agent-based-installer.html#about-the-agent-based-installer[About the Agent-based Installer].
+
+[id="ocp-4-12-agent-based-install-fips"]
+==== Agent-based installation supports installing {product-title} cluster in FIPS compliant mode
+Agent-based {product-title} Installer supports {product-title} clusters in Federal Information Processing Standards (FIPS) compliant mode. You must set the value of the `fips` field to `True` in the `install-config.yaml` file.
```

Comment:
I see that there are 5 instances of "Agent-based Installer" and 5 instances of "Agent-based {product-title} Installer". Is it possible to sync them all up on one or the other?